### DISCUSSION TITLE
Control playergroup's members in HA

### DISCUSSION THREAD

**Discussion by jpbaril** [music-assistant/support#3083] [Q&A] [answered]
I would like to be able to control in HA which players are on/off in a speaker group.

BTW, new playergroup UI and dynamic member mechanism is great !

**Comment by OzGav** [comment]
You can. Use a Temporary sync group or a permanent sync group with the dynamic member option on.

**Reply by jpbaril** [reply]
Sorry, I'll need more details.
I do have a playergroup in MA and it appears in HA, but how can I activate/deactivate a specific speaker in that group through HA automations ?

**Reply by OzGav** [reply]
https://music-assistant.io/faq/groups/ See the temporary sync group section

**Reply by jpbaril** [reply]
OK, I had not noticed this action/service : https://www.home-assistant.io/integrations/media_player/#action-media_playerjoin
Thanks

**Reply by jpbaril** [reply]
Well, join/unjoin is not really what I'm looking for as they literally add or remove a speaker from a group.
What I'm looking for is simply to activate/deactivate the speaker yet keeping it in the group, so the equivalent for the checking/unchecking the speaker checkbox in the speaker menu.
https://music-assistant.io/assets/screenshots/UI-speakers-menu.png

**Reply by OzGav** [reply, marked as answer]
When the speaker box is unchecked it is not part of the group. It is just showing you available players to add. You can mute the player.

**Reply by marcelveldt** [reply]
You can use join and unjoin against the group with dynamic mode enabled

**Reply by jpbaril** [reply]
>  It is just showing you available players to add.

And that's exactly what I would like to keep and that I don't get if I use the unjoin action.
Really, I would like to programmatically get the exact same result (sound-wise but also ui-wise) as if I manually unchecked a speaker in a dynamic group through the UI speaker panel.

So, above answer is not the definitive answer but I understand that what I'm looking for is not possible.
